libs/ddhub-client-gateway-frontend/ui/login/src/lib/LoginStatus/LoginStatus.effects.tsx

```diff
@@ -28,14 +30,26 @@ export const useLoginStatusEffects = () => {
     </LoadingInfo>
   );
 
+  const showLoginForm = () => {
+    if (isLoading) {
+      return checkingIdentity();
+    } else {
+      return <LoginForm onPrivateKeySubmit={privateKeyHandler} />;
+    }
+  };
+
+  useEffect(() => {
+    if (status === AccountStatusEnum.FirstLogin) {
+      setIsFirstLogin(true);
+    }
+  }, [status]);
+
   const statusFactory = () => {
     switch (status) {
+      case AccountStatusEnum.FirstLogin:
+        return showLoginForm();
```

Comment:
Can simplify by removing this line since it is sharing the same code block as below case.